.jshintrc

```diff
@@ -30,6 +30,7 @@
     "define": true,
     "require": true,
     "Chromath": false,
-    "setImmediate": true
+    "setImmediate": true,
+    "$": false
```

Comment:
Grunt run was complaining and threw the following error:

$ grunt
Running "jscs:src" (jscs) task
>> 155 files without code style errors.

Running "jshint:source" (jshint) task

public/app/features/dashboard/dashboardNavCtrl.js
  line 55  col 7  '$' is not defined.

✖ 1 problem

Warning: Task "jshint:source" failed. Use --force to continue.

Aborted due to warnings.


Then I searched online a bit and found that we need to enable $ as a variable. Actually, this is the link which I followed: http://stackoverflow.com/questions/8852765/jshint-strict-mode-and-jquery-is-not-defined